pkg/util/util.go

```diff
@@ -45,6 +50,14 @@ func GetDefaultClusterID() string {
 	return getEnv(defaultClusterIDVar, fallbackDefaultClusterID)
 }
 
+func GetAckWaitMinutes() int {
```

Comment:
Maybe add here comments as they are for the other methods.

Reply:
I think the comment would just explain a second time what the method name already explains. But lets wait for the PR against upstream ... may you are right :grin: